pyaedt/modeler/Modeler.py

```diff
@@ -1333,21 +1410,24 @@ def convert_to_selections(self, objtosplit, return_list=False):
 
     @aedt_exception_handler
     def split(self, objects, plane, sides="Both"):
-        """Split object list
+        """Split a list of objects.
 
         Parameters
         ----------
-        objects :
-            it contains an object or a list of objects to be. if it is a string, it is expected an object name, otherwise and object id
+        objects : str, int, or list
+            One or more objects to split. A list can contain
+            both strings (object names) and integers (object IDs).
         plane :
```

Comment:
```suggestion
        plane : int
```

@maxcapodi78, I'm seeing a bunch of these parameters that accept `CoordinateSystemPlane`, but this isn't correct.  It appears that you're expecting an int from 0 - 2, and supplying an object of `CoordinateSystemPlane` will raise an index error (at least in this method).  Can we just request an `int` and then provide a reference to `CoordinateSystemPlane` for allowable types?

If we want to ensure the user uses a CoordinateSystemPlane then we probably should be using `enum` instead of `class` in `modeler_constants`.
```
import enum

class Plane(enum.Enum):
    x = 0
    y = 1
    z = 2


>>> Plane.x
<Plane.x: 0>
```